src/utils/AFramePhysics.js

```diff
@@ -27,7 +27,7 @@ AFRAME.registerComponent("force-pushable", {
 //This set aframe entity to different layer (range from 0-31)
 //all the regular MYR entities will goes to layer 0
 //And other, such as grid, light indicator will goes to layer 1 so it won't take effect of user created light
-AFRAME.registerComponent("layer",{
```

Comment:
Renamed this line from `layer` to `lightlayer` since `layer` was already used in a new AFrame version

Reply:
Potential area that breaks light indicators